src/main/java/com/manardenza/console/ContentsUserMenu.java

```diff
@@ -0,0 +1,265 @@
+        outputSplitLine();
+        try {
+            firstName = String.valueOf(inputDataFromUser("Enter name: ", false,
+                    false, true, false));
+            lastName = String.valueOf(inputDataFromUser("Enter last name: ", false,
+                    false, true, false));
+            userController.loginUser(firstName, lastName);
+        } catch (IOException e) {
+            outputDataInConsole("Is invalid user data");
+        }
+        outputDataInConsole(firstName + ", You have successfully logged in!\n\n\t Welcome to our site!\n");
+    }
+
+    private void logoutUserMenu() throws IOException {
+        userController.logoutUser();
+        mainMenu();
+    }
+
+    private List<Hotel> findHotelByNameMenu() {
+        String hotelName = null;
+        try {
+            hotelName = String.valueOf(inputDataFromUser("Enter hotel name: ", false,
+                    false, true, false));
+        } catch (IOException e) {
+            outputDataInConsole("Entered incorrect name of the hotel");
+        }
+        outputDataInConsole("The list of found hotels by name: " + hotelName);
+        outputSplitLine();
+        List<Hotel> hotelList = hotelController.findHotelByName(hotelName);
+        hotelList.forEach(item -> outputDataInConsole(item.toString()));
+        return hotelList;
+    }
+
+    private List<Hotel> findHotelByCityMenu() {
+        String cityName = null;
+        try {
+            cityName = String.valueOf(inputDataFromUser("Enter city name: ", false,
+                    false, true, false));
+        } catch (IOException e) {
+            outputDataInConsole("Entered incorrect name of the city");
+        }
+        outputDataInConsole("List of hotels in the city found : " + cityName);
+        List<Hotel> hotelList = hotelController.findHotelByCity(cityName);
+        hotelList.forEach(item -> outputDataInConsole(item.toString()));
+        return hotelList;
+    }
+
+    private Map<String, List<Room>> findRoomMenu() throws IOException {
+        String city = null;
+        Integer person = null;
+        Integer price = null;
+        Date reservedFrom = null;
+        Date reservedTo = null;
+        try {
+            city = String.valueOf(inputDataFromUser("Enter city name: ", false,
+                    false, true, false));
+            person = (Integer) inputDataFromUser("Enter the number of persons: ", true,
+                    false, false, false);
+            price = (Integer) inputDataFromUser("Enter maximum price rate: ", true,
+                    false, false, false);
+            reservedFrom = (Date) (inputDataFromUser("Enter arrival date in the format dd.mm.yyyy: ",
+                    false, false, false, true));
+            reservedTo = (Date) (inputDataFromUser("Enter the departure date in the format dd.mm.yyyy: ",
+                    false, false, false, true));
+            compareDate(reservedFrom, reservedTo);
+        } catch (IOException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1166)